spec/requests/browse_sections_controller_spec.rb

```diff
@@ -3,8 +3,6 @@
 RSpec.describe BrowseSectionsController, type: :request do
   subject { response }
 
-  let(:json) { JSON.parse(response.body) }
```

Comment:
Unrelated to this ticket.
I just found we don't need this line 😄

Reply:
:raised_hands: